src/common/util/CMakeLists.txt

```diff
@@ -77,7 +77,8 @@ target_link_libraries(${TARGET_NAME} PRIVATE ${CMAKE_DL_LIBS} PUBLIC openvino::p
 if (WIN32)
     target_link_libraries(${TARGET_NAME} PRIVATE Shlwapi)
 endif()
-target_include_directories(${TARGET_NAME} PUBLIC $<BUILD_INTERFACE:${UTIL_INCLUDE_DIR}>)
+target_include_directories(${TARGET_NAME} PUBLIC $<BUILD_INTERFACE:${UTIL_INCLUDE_DIR}>
```

Comment:
This dependency already exist - openvino/core/visibility.hpp is included in src/common/util/include/openvino/util/file_path.hpp.
Probably definition of `OPENVINO_CPP_VER_11` and `OPENVINO_CPP_VER_17` should be moved into CMake configuration stage.

Reply:
> Probably definition of OPENVINO_CPP_VER_11 and OPENVINO_CPP_VER_17

It's compile time macro, which means on OpenVINO build side it can be of one value, on users side - another.
Even within OpenVINO we have some plugins compiled with C++17, so it may vary from where this header is included.

> There should not be dependency in this direction (no core includes) but core can use it. @ilya-lavrenov could you check confirm?

Agree.

Reply:
openvino::utils should be a common OV independent library with some C++ helpers, filesystem utils.

Reply:
Done